Source/WebCore/loader/cache/CachedResourceRequest.cpp

```diff
@@ -190,7 +194,7 @@ String CachedResourceRequest::acceptHeaderValueFromType(CachedResource::Type typ
 void CachedResourceRequest::setAcceptHeaderIfNone(CachedResource::Type type)
```

Comment:
We should also take in a `LocalFrame*`, there is a simple call site and this call site call pass `protectedFrame().get()`